src/providers/bitcoin/BitcoinRPCProvider.js

```diff
@@ -144,6 +144,22 @@ export default class BitcoinRPCProvider extends JsonRpcProvider {
     }
   }
 
+  async getUsedAddresses () {
+    const addresses = await this.jsonrpc('listaddressgroupings')
```

Comment:
As we discussed, this would have to return every address ever generated by the node.